lib/shrine/plugins/zencoder.rb

```diff
@@ -17,7 +17,7 @@ def perform data, payload
 
       module AttacherMethods
         def start_encoding data
-          ::Zencoder::Job.create input: url,
+          ::Zencoder::Job.create input: url(:original),
```

Comment:
@zpieslak Does this support the cache versions too? Or would it be better to override the `url` method from the uploader?